qiskit_nature/problems/second_quantization/electronic/builders/hopping_ops_builder.py

```diff
@@ -133,8 +133,8 @@ def _build_single_hopping_operator(
                     commutativities.append(False)
             else:
                 raise QiskitNatureError(
-                    "Symmetry {} is nor commute neither anti-commute "
-                    "to exciting operator.".format(symmetry.to_label())
+                    f"Symmetry {symmetry.to_label()} is nor commute neither anti-commute "
+                    "to exciting operator."
```

Comment:
```suggestion
                    f"Symmetry {symmetry.to_label()} is neither commutes nor anti-commutes "
                    "with excitation operator."
```
In reading the above the text is not all that great - maybe this suggestion is an improvement and is what was meant